src/main/java/io/anserini/ltr/FeatureExtractorCli.java

```diff
@@ -82,16 +82,15 @@
     if (parsedArgs.collection.equals("gov2") || parsedArgs.collection.equals("webxml")) {
       // Open the topics file and read it
       String className = parsedArgs.collection.equals("gov2") ? "Trec" : "Webxml";
-      TopicReader tr = (TopicReader)Class.forName("io.anserini.search.query."+className+"TopicReader")
+      TopicReader<String> tr = (TopicReader)Class.forName("io.anserini.search.query."+className+"TopicReader")
```

Comment:
We really need to refactor this piece of code. The generic type here should be decided by `className`.
Some classes use integers while others (only `CarTopicReader`? @Victor0118 ) use string.